config/quota-management-list-configuration.yaml

```diff
@@ -5,25 +5,62 @@
 # In the future it may support service accounts outside RH.
 # The structure of registered service accounts is:
 #       - username: is the account of the user. The username must be unique
-#       - max_allowed_instances: is the maximum number of instances this user can create.
+#       - max_allowed_instances: is the maximum number of instances of a given instance_type/billing_model pair this user can create.
+#         This value is used if `allowed` field (inside `billing_model`) is 0 or omitted.
 #         Defaults to the global value of `max-allowed-instances` which has different values for distinct environments.
+#       - granted_quota: the configuration of the quota granted to the user. The configuration is defined as follows:
+#         - instance_type_id: the id of the instance type for which we are configuring the quota
+#         - billing_models: the list of billing models that we are going to configure. This is defined as follows:
+#           - id: the id of the billing model we are going to configure
+#           - allowed: the number of instances of this type the user can create
+#           - expiration: after how many days the instance will expire (it will be suspended). Defaults to 0 (unlimited)
+#           - grace_period: after how many days after being suspended, the instance will be deleted. Defaults to 0 (unlimited)
```

Comment:
It is also a good idea to specify the units in the name of the attributes when it accepts a specific unit. I suggest expiration_\<accepted_unit\>. Same for grace_period